edx_rbac/utils.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import crum
+
+from edx_rest_framework_extensions.auth.jwt.cookies import (
+    jwt_cookie_name,
```

Comment:
why isn't this being used below to get the jwt_cookie_name?

Reply:
ah, probably a copy-paste thing when i was extracting the code from the protoytype. this function is actually just 1 line... i wonder if it's even worth importing

Reply:
i did end up making this change, so resolving!